### PR TITLE
[Typography] Converts auction pages (above the fold)

### PR DESCRIPTION
**Pull request by dzucconi**
So this is primarily dealing with the desktop/mobile above the fold auction pages, but there's some bleed into a few global things.

* The add to calendar component, which was only also used on the fair events page — tested there and looks good.
* The clock component — this one I'm not 100% on all the actual active call sites. It appears that there's a different clock that's used, for instance, on artwork pages. If someone can point out other places the CoffeeScript version lives, I'd like to double check them.
* Active bids, which is used in the auctions index and shows up in settings, in addition to the actual auction pages

----

----

----

----

----

### CODE REVIEW

**Comment by dzucconi** [comment]
Fixed up the clock display on the (soon to be discontinued) fair page:

File: src/desktop/apps/fair_organizer/stylesheets/index.styl [hunk 1]
```diff
@@ -184,10 +184,9 @@ browse-container-height = 442px
 .fair-organizer-top__countdown__clock .clock-header
   border 0
   padding 0
-  margin 20px 0
+  margin 20px 0 0 0
```
Comment (anandaroop) [inline src/desktop/apps/fair_organizer/stylesheets/index.styl:187]: nbd but could be `getSpace` as elsewhere?

**Review by anandaroop — APPROVED** [review]
Nice!

File: src/desktop/apps/fair_organizer/stylesheets/index.styl (hunk 1 above)
Comment (dzucconi) [reply src/desktop/apps/fair_organizer/stylesheets/index.styl:187]: 🥴 I didn't `require` palette here and now I'm afraid of angering the build again; we'll let it ride. 😉

**Pull request merged**